src/logs/metadata_engine/metadata_engine.py

```diff
@@ -139,15 +140,22 @@ def _apply_rule(rule, record, parsed_record):
     if rule.aws_loggroup_pattern and "log_group" in record:
         extracted_values = parse_aws_loggroup_with_grok_pattern(record["log_group"], rule.aws_loggroup_pattern)
         record.update(extracted_values)
+    if rule.log_content_parse_type in "json":
```

Comment:
"in" or "is"?

Reply:
![image](https://user-images.githubusercontent.com/50921490/122368144-5d59b380-cf5d-11eb-9dfe-0f35d6133798.png)

Reply:
But in was a bad idea, it crashes with left hand side =None